nussl/separation/nmf_mfcc.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import sklearn.cluster
+import librosa
+import nussl.audio_signal
+import nussl.constants
+import nussl.spectral_utils
+import nussl.utils
+from nussl.transformers import transformer_nmf
+from nussl.separation import separation_base
+import mask_separation_base
+import masks
+
+
+class NMF_MFCC(mask_separation_base.MaskSeparationBase):
+    """
+        Non Negative Matrix Factorization using K-Means Clustering on MFCC (NMF MFCC) is a source separation
+        algorithm that runs Transformer NMF on the magnitude spectrogram of an input audio signal.
+        It uses K means clustering to cluster the templates and activations returned by the NMF. The dot product of the
```

Comment:
What do MFCC's have to do with this algorithm? Just based on this description, I might be confused about what role MFCC's play.